back/src/Guide/Api/Methods.hs

```diff
@@ -241,6 +242,7 @@ addEdit edit = unless (isVacuousEdit edit) $ do
     Context Config{..} _ RequestDetails{..} <- ask
     dbUpdate $ RegisterEdit edit rdIp time
     dbUpdate $ RegisterAction (Action'Edit edit) rdIp time _baseUrl rdReferer rdUserAgent
+    postMatomo $ Matomo rdIp rdUserAgent rdReferer edit
```

Comment:
This should be done asynchronously, so that if Matomo is down the wiki will still work.

Also if no Matomo is configured in the config, we shouldn't send analytics at all.